FordLabs/PeopleMover
# My role warning

## Issue
Resolves #432  

## What was done
- [x] Fix creating multiple trait that look the same because of trailing space 

## How to test
1. create a role, then try to create the same role and add space at the beginning or the end or both and see that you can't 

Feature URL: https://featurebranchui.apps.pd01i.edc1.cf.ford.com/
Feature Branch Deployment: https://jenkins-fordlabs.apps.pd01e.edc1.cf.ford.com/job/PeopleMover/job/Deploy%20PeopleMover%20FeatureBranch/
